app/views/wiki/methods.html.erb

```diff
@@ -62,3 +62,4 @@
   </div>
   <hr style="clear:both;" />
 </div>
+e
```

Comment:
@Priyaraj17, this is the second file @Tlazypanda was mentioning. This change is not required.

Reply:
Hey, I have removed the changes.